examples/index.tsx

```diff
@@ -6,8 +6,15 @@ import * as React from 'react';
 import { ThemeProvider } from '@itwin/itwinui-react';
 
 const withThemeProvider = (Component: () => React.ReactElement) => () => {
+  const [portalContainer, setPortalContainer] = React.useState<HTMLElement>();
+  React.useEffect(() => void setPortalContainer(document.body), []);
```

Comment:
Is there a reason to not initialize the `portalContainer` itself? Like why not pass `document.body` to `useState()` on L9?

Does this have something to do with `document.body` not being available before `React.useEffect()` can be called? Like some SSR related quirk?

Reply:
yes, `document` only works on the client